tests/fixtures/tts.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import soundfile as sf
+import torch
+
+from nemo.collections.asr.parts.utils.manifest_utils import read_manifest
+
+
+@pytest.fixture()
+def set_device():
+    return torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+@pytest.fixture()
+def language_specific_text_example():
+    return {
+        "en": "Caslon's type is clear and neat, and fairly well designed;",
```

Comment:
Flaky if you forget to add language when you upload model

Reply:
"forget to add" will make this test failed. This is on purpose. We enforce contributors to add example sentence when they scale with new languages.